### PR TITLE
Updated terminology, modified some connection info

### PR DESCRIPTION
**Pull request by jaydestro**
modified some of the terminology as per mark:
- s/records/documents

made changes to connection details and methods of getting the code

set all to use autoscale 1000 instead of manual 400

added details for db creation in preallocation that got lost when it was rewritten.

### CODE REVIEW

File: attribute-array/README.md [hunk 1]
```diff
@@ -151,30 +151,24 @@ As you may have multiple versions of the runtime installed, make sure that .NET
...
-There are a few ways you can start working with the code in this demo.
-
 ### **Clone the Repository to Your Local Computer:**
 
 **Using the Terminal:**
 
 - Open the terminal on your computer.
 - Navigate to the directory where you want to clone the repository.
-- Type `git clone https://github.com/Azure-Samples/cosmos-db-design-patterns.git` and press enter.
+- Type `git clone git clone https://github.com/Azure-Samples/cosmos-db-design-patterns.git` and press enter.
```
Comment (markjbrown) [inline attribute-array/README.md:160]: Why is git clone here twice?

File: data-binning/README.md [hunk 2]
```diff
@@ -12,7 +12,7 @@ description: Review this case study on how to design and implement binning to op
 
 # Azure Cosmos DB design pattern: Data Binning
 
-The binning pattern (sometimes called windowing pattern) is used when the data being tracked is being created at a high velocity and the use case only requires summaries or snapshots of a window of data. A common example is if one thousand sensors are configured to each track one reading per 5 seconds. There may be an alerting system that benefits from that frequency, but the applications using Azure Cosmos DB only want to see the data summarized by minute. As data is generated, it is summarized over a window of 1 minute and only the summarized records are stored in the Azure Cosmos DB collection.
```
Comment (markjbrown) [inline data-binning/README.md:15]: The language in the last sentence doesn't look right.

File: distributed-counter/README.md [hunk 3]
```diff
@@ -99,40 +99,24 @@ You should have a version 4._x_ installed. If you do not have this version insta
...
-There are a few ways you can start working with the code in this demo.
-
 ### **Clone the Repository to Your Local Computer:**
 
 **Using the Terminal:**
 
 - Open the terminal on your computer.
 - Navigate to the directory where you want to clone the repository.
-- Type `git clone https://github.com/Azure-Samples/cosmos-db-design-patterns.git` and press enter.
+- Type `git clone git clone https://github.com/Azure-Samples/cosmos-db-design-patterns.git` and press enter.
```
Comment (markjbrown) [inline distributed-counter/README.md:108]: git clone here twice.

File: preallocation/README.md [hunk 4]
```diff
@@ -255,11 +239,22 @@ You can try out this implementation by running the code in [GitHub Codespaces](h
 
 1. Create a free Azure Cosmos DB for NoSQL account: (<https://cosmos.azure.com/try>)
 
+1. In the Data Explorer, create a new databased named **HotelDB** with shared autoscale throughput and a container **Hotels**:
```
Comment (markjbrown) [inline preallocation/README.md]: You are copy/pasting incorrect container names in here.

File: attribute-array/README.md (hunk 1 above)
Comment (jaydestro) [reply attribute-array/README.md:160]: removed git clone and just left the URL.  got it.

File: distributed-counter/README.md (hunk 3 above)
Comment (jaydestro) [reply distributed-counter/README.md:108]: same as above

File: preallocation/README.md (hunk 4 above)
Comment (jaydestro) [reply preallocation/README.md]: @markjbrown 

matched to copy what was in appsettings.json

**Pull request merged**